### ISSUE TITLE
Autovalidate error on non stable channels

### ISSUE THREAD

**Issue by jinfagang** [codegrue/card_settings#137]
```
./../../../flutter/.pub-cache/hosted/pub.flutter-io.cn/card_settings-1.13.0+hotfix.1/lib/widgets/picker_
    fields/card_settings_radio_picker.dart:15:7: Error: The non-abstract class 'CardSettingsRadioPicker' is
    missing implementations for these members:
     - ICommonFieldProperties.autovalidate
    Try to either
     - provide an implementation,
     - inherit an implementation from a superclass or mixin,
     - mark the class as abstract, or
     - provide a 'noSuchMethod' implementation.

    class CardSettingsRadioPicker extends FormField<String>
          ^^^^^^^^^^^^^^^^^^^^^^^
    ../../../../flutter/.pub-cache/hosted/pub.flutter-io.cn/card_settings-1.13.0+hotfix.1/lib/interfaces/comm
    on_field_properties.dart:19:14: Context: 'ICommonFieldProperties.autovalidate' is defined here.
      final bool autovalidate = null;
                 ^^^^^^^^^^^^
    ../../../../flutter/.pub-cache/hosted/pub.flutter-io.cn/card_settings-1.13.0+hotfix.1/lib/widgets/picker_
    fields/card_settings_selection_picker.dart:15:7: Error: The non-abstract class
    'CardSettingsSelectionPicker' is missing implementations for these members:
     - ICommonFieldProperties.autovalidate
    Try to either
     - provide an implementation,
     - inherit an implementation from a superclass or mixin,
     - mark the class as abstract, or
     - provide a 'noSuchMethod' implementation.

    class CardSettingsSelectionPicker extends FormField<String>
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ../../../../flutter/.pub-cache/hosted/pub.flutter-io.cn/card_settings-1.13.0+hotfix.1/lib/interfaces/comm
    on_field_properties.dart:19:14: Context: 'ICommonField
```

```
→ flutter doctor                                                                   ⎇  master ✘ ^#!+?|3cf94c6
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel dev, 1.22.0-12.0.pre, on Mac OS X 10.15.6 19G2021, locale en-CN)
[✗] Android toolchain - develop for Android devices
    ✗ Unable to locate Android SDK.
      Install Android Studio from: https://developer.android.com/studio/index.html
      On first launch it will assist you in installing the Android SDK components.
      (or visit https://flutter.dev/docs/get-started/install/macos#android-setup for detailed instructions).
      If the Android SDK has been installed to a custom location, set ANDROID_SDK_ROOT to that location.
      You may also want to add it to your PATH environment variable.

 
[✓] Xcode - develop for iOS and macOS (Xcode 12.0)
[!] Android Studio (not installed)
[✓] Connected device (2 available)

! Doctor found issues in 2 categories.
```

Can u help to fix it since I am not able to build with flutter master channel, also I believe it will broken up eventually when flutter upgrade version.

**Comment by ximunfroy911th** [comment]
You should use the flutter channel stable.

**Comment by jinfagang** [comment]
@ximunfroy911th thanks for your help, but your help is useless, I know it is need stable, but I am build a desktop on macOS, it needs master branch for build. Also master branch doesn't means this branch is stadalone with stable, it's the future of stable, this is why I addressed out this issue before master branch became stable.... I wish author of this lib can notice this and provide a fix for master so that someone like me can get along with it.... @codegrue

**Comment by KingLudwig94** [comment]
I'm having the same issue on beta channel, this will become stable in the next few weeks so this will become a issue for everyone

**Comment by KingLudwig94** [comment]
I have committed a PR solving this issue #138

**Comment by codegrue** [comment]
PR Merged. Testing today

**Comment by codegrue** [comment]
Can someone help test the latest checked-in code? I cannot get it to build on iOS and am wondering if it's my system...

**Comment by rubgithub** [comment]
> class CardSettingsPassword extends CardSettingsText
>       ^^^^^^^^^^^^^^^^^^^^
> /D:/flutter/.pub-cache/hosted/pub.dartlang.org/card_settings-1.13.0+hotfix.1/lib/interfaces/common_field_properties.dart:19:14: Context: 'ICommonFieldProperties.autovalidate' is defined here.
>   final bool autovalidate = null;
>              ^^^^^^^^^^^^

**Comment by KingLudwig94** [comment]
I've run my application using the code both in iOS 13.7 and in 14.0.1. I will try to run on stable channel, it has just been updated today so it should work.

**Comment by rubgithub** [comment]
I'm getting an error:

> /flutter/.pub-cache/hosted/pub.dartlang.org/card_settings-1.13.0+hotfix.1/lib/widgets/picker_fields/card_settings_date_picker.dart:15:7: Error: The non-abstract class 'CardSettingsDatePicker' is missing implementations for these members:
>  - ICommonFieldProperties.autovalidate

**Comment by robertoltrocha** [comment]
I don't know if I did what you asked.
But with the project I downloaded and the updated flutter it worked perfectly.
Steps:
I downloaded your project;
Did not run with version 1.20.4 of the flutter
I updated the flutter to the latest stable version: 1.22.0
I compiled the app
worked perfectly on an Iphone 7 plus, iOS 14.0.1 device
Xcode 12.0

![WhatsApp Image 2020-10-04 at 18 09 02](https://user-images.githubusercontent.com/30963205/95027271-a1c24f00-066d-11eb-81be-2c54208b9245.jpg)


Flutter not updated I had the message below
```
Running "flutter pub get" in card_settings...                   

Because every version of flutter_test from sdk depends on meta 1.1.8 and card_settings depends on meta ^1.2.3, flutter_test from sdk is forbidden.
So, because card_settings depends on flutter_test any from sdk, version solving failed.
pub get failed (1; So, because card_settings depends on flutter_test any from sdk, version solving failed.)
Process finished with exit code 1
```

**Comment by codegrue** [comment]
Fixed in version 1.13.0
